.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @rvm/maintainers @rvm/contributors
```

Comment:
i'll double check if mergeable has implemented team member functionality for CODEOWNERS. I suspect that this will not work as expected with mergeable currently.

Reply:
Should work https://github.com/mergeability/mergeable#approvals :)

Reply:
it's great if it works xD

Reply:
At least that's what the documentation says...

Reply:
Actually seems it does not work: https://github.com/rvm/rvm/pull/4963/checks?check_run_id=858856086

```
✔️ approvals: all required reviewers have approved
Input :
Settings : {"required":{"owners":true,"assignees":true}}
```

No owner approved yet! However maybe its reading CODEOWNERS from master and not from PR?

Reply:
hmm.. it's as i expected, passing in a team name instead of individual name requires extra code which is not implemented in mergeable yet.

Reply:
> At least that's what the documentation says...

yeah, CODEOWNERS should work for individuals for sure. we didn't account for teams.

Reply:
test comment

Reply:
@shine2lay have this been fixed on your side maybe?

Reply:
yes, this feature has been implemented